src/pipelines.js

```diff
@@ -550,6 +550,42 @@ export class TextGenerationPipeline extends Pipeline {
  * trained on NLI (natural language inference) tasks. Equivalent of `text-classification`
  * pipelines, but these models don't require a hardcoded number of potential classes, they
  * can be chosen at runtime. It usually means it's slower but it is **much** more flexible.
+ * 
+ * **Example:** Zero shot classification with `Xenova/bart-large-mnli` (one text).
+ * ```javascript
+ * let text = '17 rue Dantoni';
```

Comment:
Maybe we can use the same example used on the [Hub](https://huggingface.co/facebook/bart-large-mnli):
![image](https://github.com/xenova/transformers.js/assets/26504141/e020b0a9-3ebe-40ff-8725-43bedafa1699)?

Reply:
Sure, I let the other example as is but tell me if you want me to put something else or adapt the first one

Reply:
I think a use-case like that will be useful. 

1. Standard
"Last week I upgraded my iOS version and ever since then my phone has been overheating whenever I use your app."
[demo link](https://huggingface.co/MoritzLaurer/DeBERTa-v3-base-mnli-fever-anli?candidateLabels=mobile%2C+website%2C+billing%2C+account+access&multiClass=false&text=Last+week+I+upgraded+my+iOS+version+and+ever+since+then+my+phone+has+been+overheating+whenever+I+use+your+app.)
![image](https://github.com/xenova/transformers.js/assets/26504141/53f93ea5-313c-49a0-97ce-c9d3a03ae393)

2. Multiclass
"I have a problem with my iphone that needs to be resolved asap!!"
[demo link](https://huggingface.co/MoritzLaurer/DeBERTa-v3-base-mnli-fever-anli?candidateLabels=urgent%2C+not+urgent%2C+phone%2C+tablet%2C+computer&multiClass=true&text=I+have+a+problem+with+my+iphone+that+needs+to+be+resolved+asap%21%21)

![image](https://github.com/xenova/transformers.js/assets/26504141/476674ab-36d8-4299-b50b-9783384552c4)